graalpy/graalpy-micronaut-guide/README.md

```diff
@@ -439,6 +439,10 @@ This will start the application on port 8080.
 
 ## 7. GraalVM Native Executable
 
+*Note: Generating GraalVM native executable of Java applications that make use of Graal languages and virtual threads
```

Comment:
*Note: Generating GraalVM native executables of Java applications that make use of Graal languages and virtual threads at the same time is supported only on GraalVM for JDK 23 and later. Notably, virtual threads support is unconditionally reachable in Micronaut framework and therefore this applies also to our example.*